Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# #342 Fixes

@Gaben1025 Resolves #342.

_Address various cartography discrepancies identified in #342, excluding those that would be more efficiently fixed in a zone-wide work._
# Changelog
## Gilneas
### Blackwald has rock texture.
<details><summary></summary>

![image](https://user-images.githubusercontent.com/46384992/115468368-0dec5500-a244-11eb-9d94-0042d9cbc641.png)</details>
- Repainted Blackwald region.
- Added pine_impassable trees.
<details><summary></summary>

![image](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/assets/40177753/3c254de1-f633-4431-8c20-94007e2a4088)
</details>

## Uldum
### These 2 rivers shouldn't be a thing. They're too small so they should take place on rivers.bmp.
<details><summary></summary>

![image](https://user-images.githubusercontent.com/46384992/115624545-dba62a80-a30b-11eb-8ec2-371a742ef57f.png)</details>
- Redrew rivers.
- Repainted textures surrounding the rivers.
<details><summary></summary>

![image](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/assets/40177753/030c36af-e3df-47b4-8f50-e4c74b1c28b2)
</details>

## Ocean Isles
### Missing Trees
- Added trees to Zandalar.

<details><summary></summary>

![image](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/assets/40177753/a5d0bac5-02a3-489f-ae55-843e39b1be9a)
</details>

- Added trees to Kul'Tiras & Mechagon.

<details><summary></summary>

![image](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/assets/40177753/80ab9fde-6b6e-4828-b1e9-9b51cd80d16d)
</details>

- Added trees to Darkspear Isles.
- Redrew Darkspear Isles textures.

<details><summary></summary>

![image](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/assets/40177753/e227f411-c682-4017-8e83-9738d793a39e)
</details>

- Added trees to Isle of Giants.

<details><summary></summary>

![image](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/assets/40177753/817ec4be-074b-46a8-90d2-d8bab70989a6)
</details>

- Added trees to Oomsk.

<details><summary></summary>

![image](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/assets/40177753/6c1a6fb6-63cd-4e2e-a435-561db7acbda5)
</details>

- Added trees to Tel Abim.

<details><summary></summary>

![image](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/assets/40177753/a7869e9e-4b98-4e9e-9436-087871e3696c)
</details>

- Added trees to Echo Isles.

<details><summary></summary>

![image](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/assets/40177753/fc860f59-622b-4652-b2a2-672b342fe2a4)
</details>

- Added trees to Strand of the Ancients.

<details><summary></summary>

![image](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/assets/40177753/a38cf6f4-7410-4f61-93b6-6828e06e92a3)
</details>

# Known Issues (To Do)
The following areas require much more thorough work that will be covered in future reworks.
- Northrend
- Pandaria
- Darkshore

# How to test:
(This section shamelessly copied from ercarp)

Consider the following when giving your feedback:

**Gameplay:** For pertinent bugs, verify if the bugfixes are working as intended.

**Canonicity:** Compare the map changes to ensure accuracy according to the information made available by official canon.

**Familiarity:** Observe whether or not the changes are recognizable and fitting for their respective locations.

**Aesthetics:** Comment on the general look and appeal on the changes. Suggestions should be as specific as possible.

Additionally, ensure that necessary gameplay features like holdings are not obscured by map features such as water, or steep cliffs.

### -- @ValianBlue
